frontend/src/scenes/experiments/ExperimentResult.tsx

```diff
@@ -240,29 +240,7 @@ export function ExperimentResult({ secondaryMetricId }: ExperimentResultProps):
                     />
                 </div>
             ) : (
-                experiment.start_date && (
-                    <>
-                        <div className="no-experiment-results p-4">
-                            {!targetResultsLoading && (
-                                <div className="text-center">
-                                    <div className="mb-4">
-                                        <b>
-                                            There are no results for this{' '}
-                                            {isSecondaryMetric ? 'secondary metric' : 'experiment'} yet.
-                                        </b>
-                                    </div>
-                                    {!!experimentResultCalculationError && (
-                                        <div className="text-sm mb-2">{experimentResultCalculationError}</div>
-                                    )}
-                                    <div className="text-sm ">
-                                        Wait a bit longer for your users to be exposed to the experiment. Double check
-                                        your feature flag implementation if you're still not seeing results.
-                                    </div>
-                                </div>
-                            )}
-                        </div>
-                    </>
-                )
+                experiment.start_date && <NoResultsEmptyState />
```

Comment:
the old one handles secondary metrics too 😅, which seems reasonable, since it returns the same shape. I think worth adding the same empty state for secondary charts and change the name like in Line 251

Reply:
hmm I see, but secondary results are not calculated by the same method (they live in `secondary_experiment_result.py`, and thus we're not doing the same validation where we collect the different errors and can't render the checklist.

So I'll just display a generic empty state like "Results not available yet".

Reply:
They do though, because they call the `trend|funnel experiment result` function that does validate -> https://github.com/PostHog/posthog/blob/master/ee/clickhouse/queries/experiments/secondary_experiment_result.py#L48

Reply:
but ok to just say results not available yet for now if we can get this in 👍

Reply:
oh but we set `validate=False` for sec metrics, thus not throwing when the validation fails: https://github.com/PostHog/posthog/blob/master/ee/clickhouse/queries/experiments/secondary_experiment_result.py#L50

I'm fine with rendering a generic message, just wondering why we're doing it like this?

Reply:
oh 🤦 , oh right! It's because significance on secondaries is not that important, and if there's some data, I decided to just show it instead of waiting for all data.

But open to revisiting and actually validating here too now, given we'll have much nicer error messages and the explore button